Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -245,6 +245,16 @@ private FileSystemMetadata GetFileSystemMetadata(FileSystemInfo info)
                 if (info is FileInfo fileInfo)
                 {
                     result.Length = fileInfo.Length;
+
+                    // Issue #2354 get the size of files behind symbolic links
+                    if(fileInfo.Attributes.HasFlag(FileAttributes.ReparsePoint))
+                    {
+                        using (Stream thisFileStream = File.OpenRead(fileInfo.ToString()))
```

Comment:
```suggestion
                        using (Stream thisFileStream = File.OpenRead(fileInfo.FullName))
```

`FullName` works in my environment